docs/DeveloperGuide.md

```diff
@@ -154,6 +154,41 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Delete Meeting Feature
+
+Syntax: `delMeeting i/1`
+
+Purpose: Delete a specified `meeting` from the Meeting List in `Model`
+
+#### Implementation
+
+Usage Scenario of `delMeeting`:
+
+1) User inputs `listMeeting` to view the current meetings in the `Model`'s Meeting List with their respective indexes.
+2) User then inputs `delMeeting i/1` to delete the first meeting shown in `listMeeting`. This will evoke `Command#execute` in `LogicManager`.
+
+Below is a sequence diagram that illustrates the execution of `delMeeting` command and the interaction with `Model`.
+
+![DeleteMeetingSequenceDiagram](images/DeleteSequenceDiagram.png)
```

Comment:
is this referencing the wrong diagram?

Reply:
DeleteMeetingSequenceDiagram instead of DeleteSequenceDiagram

Reply:
regarding the diagram, i think it might be better to show the return value of Command much like in the original DeleteSequenceDiagram

Reply:
Also, I'm not sure if its necessary to include the getFilteredListMeeting for this, might be good to check with TA if we need to include every method called or only those that matters

Reply:
Yea my bad, got confused with the two diagrams.
Will update the return value.
do you think I should only show the logic side and not take about the model? Or I remove the `getFilteredListMeeting` but keep the `deleteMeeting` command?

Reply:
hmm i think its correct to show model judging from the pre-existing diagram, but i'm not sure about having the `getFilteredMeetingList` in the diagram. I think we need to check with our TA if we **must** include all the methods called, or only those that are significant in the sequence diagram.